PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,39 @@
-## Description of Work
-<!--add description of work here 🙏🏼 -->
+## Description
+<!--add a short description of work here 🙏🏼 -->
 
 ### Change List
+<!-- More detailed list of the major changes -->
```

Comment:
I propose removing `### Change List` entirely. 

Github already creates a change list panel showing what files are changed, and imo, any high level changes about what the PR does belong in the ticket and the description.

Reply:
I actually find it more powerful to put the details in github rather than the ticket.  
1. I've never worked at a place that kept the same ticketing system, but github has always been a constant.  If it is in the PR, we will have the details for ever.
2. Putting any details needed in github also helps to review the PR more quickly and allows for more specific explanations about changes using links to specific changes in the PR, extra changes that were added like updating a version to fix a related issue, etc

Reply:
Yea, its always a battle around good PR hygiene.  The only time I've seen good PR descriptions always filled out was by using a bot/app that scanned the text and would pass/fail checks if things were/weren't added.  A little more forced than encouraged 😬

Reply:
Also, happy to make changes in the future as we find what is and isn't working.